packages/client/src/helpers.ts

```diff
@@ -10,7 +10,15 @@ import { Replace } from '@temporalio/common/lib/type-helpers';
 import { optionalTsToDate, requiredTsToDate } from '@temporalio/common/lib/time';
 import { decodeMapFromPayloads } from '@temporalio/common/lib/internal-non-workflow/codec-helpers';
 import { temporal, google } from '@temporalio/proto';
-import { RawWorkflowExecutionInfo, WorkflowExecutionInfo, WorkflowExecutionStatusName } from './types';
+import {
+  CountWorkflowExecution,
+  CountWorkflowExecutionsAggregationGroup,
+  RawCountWorkflowExecutions,
```

Comment:
Don't import `RawCountWorkflowExecutions*` from `./types.ts`. Instead, use the type as `temporal.api....CountWorkflowExecution*`.

Reply:
Done.